src/plugins/visualizations/public/embeddable/disabled_lab_visualization.tsx

```diff
@@ -29,28 +29,35 @@
  */
 
 import { FormattedMessage } from '@osd/i18n/react';
-import React from 'react';
+import { EuiEmptyPrompt, EuiLink } from '@elastic/eui';
+import React, { Fragment } from 'react';
 
 export function DisabledLabVisualization({ title }: { title: string }) {
   return (
     <div className="visDisabledLabVisualization">
-      <div
-        className="kuiVerticalRhythm visDisabledLabVisualization__icon kuiIcon fa-flask"
-        aria-hidden="true"
+      <EuiEmptyPrompt
+        iconType="database"
+        titleSize="s"
+        data-test-subj="visDisabledLabVisualization"
+        title={
+          <FormattedMessage
+            id="visualizations.disabledLabVisualizationTitle"
+            defaultMessage="{title} is an experimental visualization."
+            values={{ title: <em className="visDisabledLabVisualizationtitle">{title}</em> }}
+          />
+        }
+        body={
+          <Fragment>
+            <p>
+              Enable experimental visualizations within{' '}
+              <EuiLink href="https://github.com/app/management/opensearch-dashboards/settings">
```

Comment:
Using a url directly seems not the best way to do this? According to https://github.com/opensearch-project/OpenSearch-Dashboards/issues/3208, doc links should be from `DocLinksService`. Some places you could check:
* src/core/public/doc_links/doc_links_service.ts: could add a section`management: { advancedSettingsLink: "**"}` for Dashboards Management links
* src/plugins/visualizations/public/services.ts: import `DocLinksStart` and add `export const [getDocLinks, setDocLinks] = createGetterSetter<DocLinksStart>('docLinks');`
* src/plugins/visualizations/public/embeddable/disabled_lab_visualization.tsx: use `getDocLinks` to get the link and use it in href.

The links seems not correct to me as well. `https://github.com/app/management/opensearch-dashboards/settings` is 404. I think this `https://opensearch.org/docs/latest/dashboards/management/advanced-settings/` might be the one if you are trying to direct people to instructions on how to use advanced settings.

Reply:
Thank you! I wasn't completely sure about what I was supposed to link to. I'll make the appropriate changes.